third_party/git-cmake-format/CMakeLists.txt

```diff
@@ -1,16 +1,15 @@
 message(STATUS "Fetching git-cmake-format")
 include(FetchContent)
 FetchContent_Declare(
-    git_cmake_format
-    GIT_REPOSITORY https://github.com/ginkgo-project/git-cmake-format.git
-    GIT_TAG        e9a82f20d36f1a92d4c52910a8d0a694e7673c54
+        git_cmake_format
+        GIT_REPOSITORY https://github.com/ginkgo-project/git-cmake-format.git
+        GIT_TAG customize-toplevel-dir
```

Comment:
I assume this is the reason why it is marked as `WIP` (meaning you will revert this back to either the new checksum or `master`)?

Reply:
Exactly, but I wanted to allow testing it in #1223